test/parallel/test-timers-clear-timeout-interval-equivalent.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+
+// This test makes sure that timers created with setTimeout can be disarmed by
+// clearInterval and that timers created with setInterval can be disarmed by
+// clearTimeout.
+//
+// This behavior is documented in Node.js' documentation and the HTML Living
+// standard:
+//
+// * Refs: https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-setinterval
+// * Refs: https://nodejs.org/api/timers.html#timers_class_timeout
+
+const assert = require('assert');
+const Timer = require('timers');
+
+let nbTimersTriggered = 0;
+
+// Disarm interval with clearTimeout.
+const interval = Timer.setInterval(() => {
+  nbTimersTriggered += 1;
+}, common.platformTimeout(20));
+Timer.clearTimeout(interval);
+
+// Disarm timeout with setInterval.
+const timeout = Timer.setTimeout(() => {
+  nbTimersTriggered += 1;
+}, common.platformTimeout(20));
```

Comment:
`common.platformTimeout` isn't necessary. You can also just use `1` for the timeout value since it shouldn't execute regardless.